packages/docs/src/components/home/ComponentGallery.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <v-responsive class="py-16">
+    <v-container class="pt-0">
+      <HomeBgGradient />
+
+      <h4 class="text-h4 font-weight-bold mb-5">
+        Component Gallery
+      </h4>
+
+      <v-responsive class="mx-auto mb-5" max-width="700">
+        <h6 class="text-h6 font-weight-regular text-medium-emphasis">
+          Vuetify provides a comprehensive collection of components that can be used to build your application.
+        </h6>
+      </v-responsive>
+
+      <div class="d-flex justify-center mb-10">
+        <v-chip-group
```

Comment:
I think we should add the `mandatory` prop here so something is always selected, IMO it looks funny when nothing is selected